apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/entity/state/derived/DerivedEntity.java

```diff
@@ -13,7 +14,7 @@
 import java.util.Objects;
 
 @ToString
-public abstract class DerivedEntity implements Comparable<DerivedEntity>, Cloneable {
+public abstract class DerivedEntity implements Comparable<DerivedEntity>, Cloneable, Searchable {
```

Comment:
Looks like it is not obvious to use the same interface for both entities and dao

Reply:
Now it is used for Entity only.